css/style.css

```diff
@@ -57,49 +60,66 @@ h1,h2,h3,h4,h5 {
     fill: #f5c84266 !important;
 }
 .visavail .ypercentage {
-    paint-order: stroke;
+    /* paint-order: stroke;
```

Comment:
@tanganellilore - if this block is commented out, should we just delete it?

Reply:
yes, class can be remove if actual version is ok.
Also on index.html is present a commented old code, but I prefer, before delete some code,  maintain commented if some rollback will be required

Reply:
When everything will be approved, I proceed with deletion of comment line